tekton-example/README.md

```diff
@@ -0,0 +1,64 @@
+# Tekton Pipeline/Task Example  
+
+A small example of using `promote` in a Tekton Pipeline to promote a service's config to a GitOps repository.  Creation of the
+PipelineRun (using `service-promote-pipeline-run.yaml`) will drive the pipeline to clone, build and push the service and then promote the config from the local clone into your staging/test GitOps repository.
+
+Creation of a TaskRun (using `promote-run.yaml`) will then further promote from one GitOps repo to another, e.g. from
+staging to production.
+
+## Template Files
+
+- `auth.yaml`: Creates secrets for GitHub and Docker registry and Create the ServiceAccount
+- `gitconfig`: Data file for the configmap
+- `promote-secret.yaml`: Creates an access token secret for the GitHub repository
+- `resources.yaml`: Creates PipelineResources for GitHub and Docker repositories
+- `promote.yaml`: Creates a pull request from one repository to another repository
+
+## Other Files
+- `service-promote.yaml`: Creates a promote task which promotes from service repo to env repo
+- `service-promote-pipeline.yaml`: Creates a pipeline that executes build, push and promote
```

Comment:
`Creates a pipeline that executes build, push and promote` needs TLC around the wording. Let's use this example to make it really clear what's going on